crates/hotshot-state-prover/src/circuit.rs

```diff
@@ -199,6 +246,14 @@ mod tests {
         let keys = key_pairs_for_testing();
         let st = stake_table_for_testing(&keys);
 
+        let lightclient_state = LightClientState {
```

Comment:
Instead of using 0 and F::default(), maybe use some non-default values.

Reply:
They are irrelevant right now. I can do it in the next PR.